hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitsFilters.tsx

```diff
@@ -124,8 +121,12 @@ export const OrgUnitFilters: FunctionComponent<Props> = ({
         }
         const newFilters: Record<string, unknown> = {
             ...filters,
-            [key]: value,
         };
+        if ((!value || value === '') && newFilters[key]) {
```

Comment:
nitpick : if we ever have a filter that returns a boolean, we'll be in trouble. Though if I remember correctly other places in Iaso, there's a fair chance it will return a string version like `'true'` or `'false'`